codes/csharp/chapter_tree/array_binary_tree.cs

```diff
@@ -29,27 +24,27 @@ public class ArrayBinaryTree {
     }
 
     /* 获取索引为 i 节点的左子节点的索引 */
-    public int Left(int i) {
```

Comment:
Adding static will make it inconsistent with other languages. Is there any benefit to using a static function here?

Same as others.

Reply:
revert